.github/workflows/go_test.yml

```diff
@@ -0,0 +1,23 @@
+name: Go Test
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out
+      uses: actions/checkout@v3
+
+    # Go uses the toolchain version in go.mod since 1.21
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '>=1.21.6' 
```

Comment:
Please read the Go version from the mod file: https://github.com/Jigsaw-Code/outline-sdk/blob/52ed380b366a3c21b59c04a7bbd109027f4c705e/.github/workflows/test.yml#L31

Reply:
done